glusterapi/bitrot.py

```diff
@@ -0,0 +1,66 @@
+import httplib
+
+from glusterapi.common import BaseAPI
+from glusterapi.exceptions import GlusterApiInvalidInputs
+
+
+class BitrotApis(BaseAPI):
+    def bitrot_enable(self, volume=""):
+        """
+        Gluster enable bitrot for a volume.
+
+        :param volume: (string) volume name
+        :raises: GlusterApiError or GlusterApiInvalidInputs on failure
+        """
+        volume = volume.strip()
```

Comment:
Since we validate volumename in all the functions we can move the validation to a function and raise AttributeError if the type is not string.